rsshfs

```diff
@@ -55,6 +67,6 @@ else
     rm -f "$fifo"
     mkfifo -m600 "$fifo" &&
     < "$fifo" /usr/lib/openssh/sftp-server "${sftpargs[@]}" |
-      ssh "$rhost" sshfs -o slave ":$qlpath" "$qrpath" "$qall" > "$fifo"
+      ssh $identity "$rhost" sshfs -o slave ":$qlpath" "$qrpath" "$qall" > "$fifo"
```

Comment:
Thank you for your PR.

However, I am a bit reluctant to add a specific option for identity here.
In that case, why not also adding port or the other ssh parameters?

Or maybe we should provide a generic way to pass any ssh parameter (but the remaining command line parameters are already passed to `sshfs` through `$qall`, maybe we should change this I don't know).

For now, I prefer to pass identity through `.ssh/config`:
~~~
Host myhost
    Hostname …
    IdentityFile ~/.ssh/id_something
~~~

Then:

    rsshfs folder myhost:folder